src/pages/assessments/components/Question.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import RateButtons from "./RateButtons";
+
+const Question = ({
+    questionNum,
+    question,
+    answer,
+    setAnswer,
+    text,
+    setText,
+}) => {
+    return (
+        <div>
```

Comment:
I would probably consider adding a class name to this wrapping div called `question-container`